microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -751,5 +772,36 @@ See also:
 * [Pause code with breakpoints](../javascript/breakpoints.md)
 
 Status:
-*  This checkbox is present in Microsoft Edge Canary 109.
-*  This checkbox is present in Microsoft Edge Stable 107.
+*  This checkbox is present in Microsoft Edge Canary 111.
+*  This checkbox is present in Microsoft Edge Stable 108.
+
+
+<!-- ====================================================================== -->
+## Enable Preloading Status Panel in Application panel
+
+_new as of Microsoft Edge 110_
+
+This checkbox is related to a new feature: pre-rendering pages for instant page navigations.  In certain cases (either when the browser decides, or if the page uses a special API), the next page that the user will visit is pre-rendered in the background, in order to speed up page navigation.
+
+This new panel will eventually display the list of pages that got pre-rendered by the browser and why.  This way, you can debug your code that uses this new pre-render special API.
```

Comment:
```suggestion
In certain situations, for example when using the Speculation Rules API, Microsoft Edge prefetches and prerenders the next page a user will visit to display that page faster.

This experiment enables the **Prefetching & Prerendering** section in the **Application** Tool. The new section displays the list of pages that the browser prefetched and prerendered and why, which gives you a way to debug your code.

To learn more about prerendering pages, see [Prerender pages in Chrome for instant page navigations](https://developer.chrome.com/blog/prerender-pages/).
```

That being said, I have to check if the prerender API is enabled in Microsoft Edge too.

Reply:
replaced the two paras by comment's text.  
deleted See Also same as "To learn more", tho See Also provides nice uniformity